wemod_launcher/core_nodeps.py

```diff
@@ -31,11 +32,11 @@ def check_dependencies(requirements_file: str) -> bool:
             try:
                 importlib.import_module(package)
             except ImportError:
-                from coreutils import log
+                from .core_utils import log
 
                 log(f"Package '{package}' is missing")
                 ret = False
-    return ret
+    return True
```

Comment:
I assume this is for testing,
You can replace this with a other check if importlib wont work for you.

Reply:
It was, and this file will be refactored anyway. Apologies, I forgot I left that there. I'll revert it for now.